test/TestExamples.cxx

```diff
@@ -157,8 +157,8 @@ BOOST_AUTO_TEST_CASE(Example)
       // Putting & getting
       conf->put<int>(myKey, myValue);
       // Getters return a boost::optional that will be empty if the value is not present
-      int getValue = conf->get<int>(myKey).value();
-      int getValueWithDefault = conf->get<int>("/this/key/does/not/exist").value_or(-1);
+      conf->get<int>(myKey).value();
```

Comment:
Shouldn't the return value be checked via BOOST_CHECK or something similar?

Reply:
Not really as this file contains examples (I don't know why examples are in the `test/` dir). The full tests are in the `TestConfiguration.cxx` file.